src/components/apps/launch/messages.js

```diff
@@ -0,0 +1,81 @@
+import DataI18N from "../../data/messages";
+
+const {
+    validationEmptyDiskResourceName,
+    validationDiskResourceName,
+    validationInvalidCharacters,
+} = DataI18N.messages;
+
+const helpMsgResourceRequirements = `These Resource Requirements may be adjusted so that the analysis is submitted with requests for larger minimum resources.
+Note that this may cause the analysis to wait longer in the submission queue until a node that matches those minimum requirements becomes available.
+So generally it is advisable to leave the resource requests set to the defaults that the tool or app integrator has set.`;
+
+const hpcAppWaitTimes = `This application runs on XSEDE, the U.S. National Supercomputing Network.
+Analyses run on this system enter a queue that includes many non-CyVerse users.
+Wait times for your analyses (jobs) to submit and run may be up to several days for large-memory applications.
+Please do not resubmit your analysis.
+If the status of your analysis has not changed from Submitted to Running after several days, contact
+<a href="mailto:support@cyverse.org">support@cyverse.org</a>.
+To learn more about HPC apps, see
+<a href="https://pods.iplantcollaborative.org/wiki/x/MYOdAQ" target="_blank" rel="noopener">Using HPC Apps in the DE</a>.
+<br/>
+Apply for your own XSEDE allocation to gain access to additional resources at
+<a href="https://portal.xsede.org/allocation-request-steps" target="_blank" rel="noopener">XRAS - Submit Allocation Request</a>.
+Access to this powerful system is made available to CyVerse users at no cost through a grant from the National Science Foundation.`;
+
+export default {
+    locales: "en-us",
+    messages: {
+        analysisInfo: "Analysis Info",
+        analysisName: "Analysis Name",
+        analysisParameters: "Analysis Parameters",
+        appDeprecated:
+            "This application has been deprecated." +
+            " If you need access to it, please contact" +
+            ' <a href="mailto:support@cyverse.org">support@cyverse.org</a>.',
+        appParamsDeprecated:
+            "This application uses 1 or more parameters that have been deprecated." +
+            " If you still need access to this application, please contact" +
+            ' <a href="mailto:support@cyverse.org">support@cyverse.org</a>.',
+        appUnavailable: "This app is temporarily unavailable.",
```

Comment:
We should display intercom button (first check intercom availability using `useIntercom` context) for contacting support whenever possible. That way, they get queued for our team. Emails to `support@cyverse.org` will go to `general` queue in intercom and can get lost.

Reply:
This message was just copied from the current UI, but I’ll look into updating it. 👍🏼

Reply:
I'll have to save this for another PR, because it looks like our Intercom config is setup to detect our custom Intercom launcher by ID, and there should only be one element with that ID in the DOM:
https://github.com/cyverse-de/sonora/blob/94991edb75081c6cebb2d3c051c98513d6d88204/src/pages/_app.js#L34
Maybe we can refactor it to use some other selector 🤔

Reply:
Please see https://github.com/cyverse-de/sonora/blob/2ed058ec05473e90dfe172c2fee177f92d893910/src/components/utils/error/ErrorHandler.js#L91 on how to display a button to intercom support button. That reminds me that I need to check if intercom is enabled.

Reply:
Ah, thanks!

> That reminds me that I need to check if intercom is enabled here as well.

In that case, I'll create a `showIntercom` function in `/src/common/intercom.js`, then I'll refactor `ErrorHandler` to use that as well 👍